apps/i18n/common/en_us.json

```diff
@@ -377,7 +377,7 @@
   "calendar": "Calendar",
   "theCalendar": "the calendar",
   "calendarNotAvailable": "The calendar isn't available for this course.",
-  "calendarLegacyMessage": "This class section is assigned a legacy version of {courseName}, which doesn't support the calendar. To access the calendar for this course, assign a version from '20-'21 or later.",
```

Comment:
@dju90 can you confirm that these numbers are correct, when I look at 2020 of CSP and CSD they are still in the `curriculum builder` side of things.

Reply:
Ah, looking through [Slack history](https://codedotorg.slack.com/archives/C06ERUABG01/p1725475157857889), looks like we discovered that '20-'21 curriculum and earlier don't have the calendar - so this message should say "...assign a version from '21-'22 or later"